### ISSUE TITLE
zos create does not work in a repo named '*contracts' (v 2.3.0)

### ISSUE THREAD

**Issue by JirkaChadima** [zeppelinos/zos#927]
Hi, thanks for the great work. I've stumbled upon a very funny bug in **2.3.0** which got me stuck for a few hours:

If you clone a minimal repro repo on https://github.com/JirkaChadima/zos23bugcontracts with

```
git clone git@github.com:JirkaChadima/zos23bugcontracts.git
cd zos23bugcontracts
npm install
```

and then run a `test.sh` script (against a dev net on 8545, it creates a fresh zos app and tries to upload a proxied contract), you can observe a `Cannot read property 'absolutePath' of undefined` error. This does not happen with zos `2.2.0` (There's a `zos22` branch which behaves as expected).

**What's the funny thing is, that this error does not occur, if you rename the directory from `zos23bugcontracts` to something that does not end on `contracts`**.

I'm running this on 
```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.2 LTS"
```

**Comment by jcarpanelli** [comment]
Hello, @JirkaChadima! It's indeed an ugly (but yeah, funny :joy: ) bug. We have just identified why is happening though, and will probably fix it soon! Thanks for reporting it :muscle:.

**Comment by spalladino** [comment]
Fixed in #928. We'll add this to a 2.3.1 and release in the upcoming days!

**Comment by spalladino** [comment]
@JirkaChadima we've just released 2.3.1 with a fix. Mind giving it a shot?

**Comment by JirkaChadima** [comment]
@spalladino That seems to be working, thank you!
